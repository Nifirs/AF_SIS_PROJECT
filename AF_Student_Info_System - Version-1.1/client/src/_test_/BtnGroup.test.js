import React from 'react';
import ButtonGrp from '../components/BtnGroup';
import { create } from 'react-test-renderer';

describe('My snapshot test for buttons',()=>{
    test('testing app button', () => {
    let tree = create(<ButtonGrp />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})