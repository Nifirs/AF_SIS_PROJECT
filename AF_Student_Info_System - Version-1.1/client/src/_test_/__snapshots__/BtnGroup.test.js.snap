// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My snapshot test for buttons testing app button 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#48494F",
      "width": "295px",
    }
  }
>
  <nav
    className="navbar navbar-expand navbar-dark bg-primary"
  >
    <span
      className="navbar-brand"
    >
      Important Links
    </span>
  </nav>
  <nav
    className="navbar navbar-expand navbar-light bg-light"
  >
    <a
      className="navbar-brand"
      href="http://www.moe.gov.lk/"
    >
      Ministry of Education
    </a>
  </nav>
  <nav
    className="navbar navbar-expand navbar-light bg-info"
  >
    <a
      className="navbar-brand"
      href="http://www.transport.gov.lk"
    >
      Ministry of Transport
    </a>
  </nav>
  <nav
    className="navbar navbar-expand navbar-light bg-danger"
  >
    <a
      className="navbar-brand"
      href="http://www.srilanka.travel"
    >
      Tourist Board
    </a>
  </nav>
  <nav
    className="navbar navbar-expand navbar-dark bg-secondary"
  >
    <a
      className="navbar-brand"
      href="http://www.srilankan.lk"
    >
      SL Airlines
    </a>
  </nav>
  <br />
  <div
    className="btn-group-vertical"
    role="group"
    style={
      Object {
        "height": "450px",
        "marginLeft": "10px",
      }
    }
  >
    <button
      className="btn btn-primary btn-block mt-4 btn btn-primary"
      disabled={false}
      type="button"
    >
      ABOUT
    </button>
    <button
      className="btn btn-primary btn-block mt-4 btn btn-primary"
      disabled={false}
      type="button"
    >
      CONTACT
    </button>
    <button
      className="btn btn-primary btn-block mt-4 btn btn-primary"
      disabled={false}
      type="button"
    >
      FAQ
    </button>
    <button
      className="btn btn-primary btn-block mt-4 btn btn-primary"
      disabled={false}
      type="button"
    >
      POLICIES
    </button>
    <button
      className="btn btn-primary btn-block mt-4 btn btn-primary"
      disabled={false}
      type="button"
    >
      NOTICES
    </button>
    <button
      className="btn btn-primary btn-block mt-4 btn btn-primary"
      disabled={false}
      type="button"
    >
      SERVICES
    </button>
  </div>
</div>
`;
